### PR TITLE
POM-75 Add the szukaj-pomocy/transport page

### PR DESCRIPTION
**Pull request by mgol**
Some styling & logic issues as well as unit tests still to be fixed.

### CODE REVIEW

File: src/app/find-help/transport-search/transport-search.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      [title]="result.title"
+      [description]="result.description"
+      [attributes]="[
+        { test: !!result.transportDate, text: result.transportDate + '', icon: 'date_range' },
+        {
+          test: !!result.capacity,
+          text: result.capacity + ' ' + this.pluralizePolish('osoba', 'osoby', 'osób', result.capacity),
+          icon: 'person'
+        }
+      ]"
```
Comment (sfc-gh-krzymkowski) [inline src/app/find-help/transport-search/transport-search.component.html:20]: I'm starting to think, my idea of using attributes parameters wasn't a good one. 
We should refactor it to:
```
<app-search-result ....>
 <app-search-attribute icon='date_range' *ngIf="result.transportDate">{{result.transportDate}}</app-search-attribute>
...
</...>```
What do you think?

File: src/app/find-help/transport-search/transport-search.component.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  styleUrls: ['./transport-search.component.scss'],
+})
+export class TransportSearchComponent {
+  results: TransportOffer[] = [];
+  total?: number = undefined;
+  loading = false;
+  constructor(private http: HttpClient, private transportResourceService: TransportResourceService) {}
+
+  // See https://github.com/mmiszy/polish-plurals/blob/5d2dc0cdea93e68e0a320a0201f0acb7a53206d0/index.mjs
+  pluralizePolish(one: string, few: string, many: string, count: number) {
```
Comment (sfc-gh-krzymkowski) [inline src/app/find-help/transport-search/transport-search.component.ts:17]: An angular pipe would be more efficient AFAIK, as in contrary to method call, it's executed only when input changes.

File: src/app/find-help/transport-search/transport-search.component.html (hunk 1 above)
Comment (mgol) [reply src/app/find-help/transport-search/transport-search.component.html:20]: @sfc-gh-krzymkowski makes sense to me, that's much clearer and more idiomatic.

File: src/app/find-help/transport-search/transport-search.component.ts (hunk 2 above)
Comment (mgol) [reply src/app/find-help/transport-search/transport-search.component.ts:17]: Makes sense, I'll update the code.

File: src/app/find-help/transport-search/transport-search.component.html (hunk 1 above)
Comment (mgol) [reply src/app/find-help/transport-search/transport-search.component.html]: Done.

**Comment by sfc-gh-krzymkowski** [comment]
Fantastic! Thanks @mgol!

**Pull request merged**